Apps/WebClient/src/ClientApp/src/constants/eventMessageName.ts

```diff
@@ -1,4 +1,3 @@
 export enum EventMessageName {
-    TimelineCurrentDateUpdated = "timelineCurrentDateUpdated",
-    CalendarCurrentDatesUpdated = "calendarCurrentDatesUpdated",
+    CalendarMonthChanged = "calendarMonthChanged",
```

Comment:
what are the different between ..Changed vs ...Updated ? I'm seeing that we are having "update:..." in some existing events, are we going to refactor them to use "change:..." instead?

Reply:
sure, updated it is.